x-pack/solutions/security/plugins/security_solution/common/api/entity_analytics/entity_store/common.schema.yaml

```diff
@@ -92,6 +92,8 @@ components:
                 type: string
               message:
                 type: string
+              metadata:
+                type: object
```

Comment:
Added this field to retrieve metadata related to entity store components, such as transform status